packages/clay-css/src/scss/cadmin/components/_forms.scss

```diff
@@ -178,6 +178,12 @@ label {
 		width: 0;
 	}
 
+	// CKEditor markup work around
+
+	&[contenteditable] p {
+		margin-bottom: 0;
+	}
```

Comment:
Hey @pat270, this is a little strange to keep on Clay's side, in that case wouldn't it be better to keep it in the CKEditor skins we have in DXP? I'm not sure if there were any limitations

Reply:
@matuzalemsteles This is a good point. I'm not sure where the right place would be. This markup is generic enough that it makes sense to include it in Clay. Clay is adding the margin so makes sense to remove here? :shrug:

Reply:
I imagine that despite being generic we are only covering for the case of CKEditor, as in the comment above this caught my attention 😅, probably if it's something very specific for CKEditor and Clay to work together I think this should live in the skins of CKEditor, I'm not sure if the `mono-lexicon` skins are in DXP or ckeditor's repo. But if you think this shouldn't be an issue for non-CKEditor cases we can go with that.

Reply:
I just remembered, CKE does take care of that. Cadmin increased the specificity of our styles and overrides the CKE styles. We should keep it here.

Reply:
Hmm, Just for me to understand, CKE adding some margin, is this intentional for some use cases? For example, aren't all CKE implementations using `form-control`, or just a few? if only a few use it, it might make sense to keep it here. Since without the `form-control` the CKE must need the margin.

Reply:
Browsers add `margin-bottom: 1em` by default. I believe our version of CKE uses `form-control`? I doubt they would use Bootstrap classes by default unless it was a coincidence. But I agree lets just keep it here.

Reply:
Oh, I see, okay. Let's get on with it!